src/NIST-800-53/rules/dms/nist800053DMSReplicationNotPublic.ts

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+import { CfnReplicationInstance } from '@aws-cdk/aws-dms';
+import { CfnResource, Stack } from '@aws-cdk/core';
+
+/**
+ * DMS replication instances are not public - (Control ID: AC-4)
+ * @param node the CfnResource to check
+ */
+export default function (node: CfnResource): boolean {
+  if (node instanceof CfnReplicationInstance) {
+    const publicAccess = Stack.of(node).resolve(node.publiclyAccessible);
+    if (publicAccess) {
```

Comment:
According to [Cfn Documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-dms-replicationinstance.html#cfn-dms-replicationinstance-publiclyaccessible) if this value is `undefined` then public access in enabled by default
```suggestion
    if (!(publicAccess === false) ) {
```